src/fiatExchanges/ExchangeQR.tsx

```diff
@@ -0,0 +1,191 @@
+    ValoraAnalytics.track(FiatExchangeEvents.external_exchange_qr_share, {
+      flow,
+    })
+    dispatch(shareQRCode(qrSvgRef.current))
+  }
+
+  const onPressCopy = () => {
+    ValoraAnalytics.track(FiatExchangeEvents.external_exchange_qr_copy_address, {
+      flow,
+    })
+    Clipboard.setString(address || '')
+    Logger.showMessage(t('addressCopied'))
+  }
+
+  const onPressInfo = () => {
+    ValoraAnalytics.track(FiatExchangeEvents.external_exchange_qr_bottom_sheet_open, {
+      flow,
+    })
+    setBottomSheetVisible(true)
+  }
+
+  const onPressExchange = (exchange: ExternalExchangeProvider) => {
+    ValoraAnalytics.track(FiatExchangeEvents.external_exchange_qr_bottom_sheet_link_press, {
+      flow,
+      exchange: exchange.name,
+    })
+  }
+
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      headerRight: () => (
+        <TopBarIconButton testID="shareButton" icon={<Share />} onPress={onPressShare} />
+      ),
+    })
+  }, [navigation])
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.qrContainer}>
+        <StyledQRCodeDisplay qrSvgRef={qrSvgRef} />
+      </View>
+
+      {displayName && (
+        <Text
+          style={[styles.name, fontStyles.displayName]}
+          numberOfLines={1}
+          ellipsizeMode="tail"
+          testID="displayName"
+        >
+          {displayName}
+        </Text>
+      )}
+      <Text testID="address" style={[fontStyles.mediumNumber, fontStyles.regular, styles.address]}>
+        {address}
+      </Text>
+      <Button
+        text={t('fiatExchangeFlow.exchange.copyAddress')}
+        onPress={onPressCopy}
+        icon={<Paste color={colors.white} />}
+        iconPositionLeft={false}
+        testID="copyButton"
+      />
+
+      <View style={styles.infoWrapper}>
+        <Text style={[fontStyles.regular, styles.exchangeText]}>
+          {t('fiatExchangeFlow.exchange.informationPartOne')}
+          <Text
+            testID="bottomSheetLink"
+            style={[fontStyles.regular600, styles.link]}
+            onPress={onPressInfo}
+          >
+            {t('fiatExchangeFlow.exchange.informationPartTwo')}
+          </Text>
+          {t('fiatExchangeFlow.exchange.informationPartThree')}
+        </Text>
+      </View>
+      <ExchangesBottomSheet
+        isVisible={!!bottomSheetVisible}
+        onClose={onCloseBottomSheet}
+        onExchangeSelected={onPressExchange}
+        exchanges={exchanges}
+      />
+    </View>
+  )
+}
+
+ExchangeQR.navigationOptions = ({
+  route,
+}: {
+  route: RouteProp<StackParamList, Screens.ExchangeQR>
+}) => ({
+  ...emptyHeader,
+  headerLeft: () => (
+    <BackButton
+      eventName={FiatExchangeEvents.external_exchange_qr_back}
+      eventProperties={{ flow: route.params.flow }}
+    />
+  ),
+  headerTitle:
+    route.params.flow === CICOFlow.CashIn
+      ? i18n.t(`fiatExchangeFlow.cashIn.depositExchangeTitle`)
+      : i18n.t(`fiatExchangeFlow.cashOut.withdrawExchangeTitle`),
```

Comment:
Does this just exist so we handle this case as well and won't need a new version? For the exp, we'd just have cash in navigate to this screen right?